image/build_windows_containerdisk.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$1" ]; then
+  echo "Please pass Windows image version to build as first argument."
+  exit 1
+fi
+
+if [ -z "$2" ]; then
+  echo "Please pass path to Windows ISO as second argument."
+  exit 1
+fi
+
+WINDOWS_VERSION=$1
+WINDOWS_ISO=$2
+case "$WINDOWS_VERSION" in
+  windows10)
+    OS_VARIANT=win10
+    VIRT_INSTALL_EXTRA_ARGS=(--boot uefi)
+    ;;
+  windows11)
+    OS_VARIANT=win11
+    ;;
+  windows2k16)
+    OS_VARIANT=win2k16
+    # Create BIOS bootable ISO
+    WINDOWS_BOOT_IMAGE=(-b boot/etfsboot.com)
+    ;;
+  windows2k19)
+    OS_VARIANT=win2k19
+    # Create BIOS bootable ISO
+    WINDOWS_BOOT_IMAGE=(-b boot/etfsboot.com)
+    ;;
+  windows2k22)
+    OS_VARIANT=win2k22
+    ;;
+  *)
+    echo "Need a valid Windows image version to build: windows10, windows11"
```

Comment:
maybe add windows2k16, windows2k19, etc. in this error message?

Reply:
Missed that, fixed.